src/js/api.js

```diff
@@ -8,23 +8,55 @@ function makeResponder(cb){
   }
 }
 
+function getHandler(url, cb, suffix){
+  if(typeof url === 'function'){
+    cb = url;
+    url = makeUrl(parseLocation(), suffix);
+  }
+
+  superagent.get(url).end(makeResponder(cb));
+}
+
+function postHandler(url, cb, suffix){
+  if(typeof url === 'function'){
+    cb = url;
+    url = makeUrl(parseLocation, suffix);
+  }
+
+  superagent.post(url).end(makeResponder(cb));
+}
+
+function makeUrl(obj, suffix){
```

Comment:
Err you mean only function outside of the `get` and `post` functions? It has it's own entry in `module.exports`, see https://github.com/cfpb/hmda-platform-ui/pull/105/files#diff-3d4fce1ac7e3fbf5016b8881ccffd7bcR61